integration/test/cloudwatchlogs/resources/write_logs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+
+n="$1"
+if [ -z "$1" ] || [ $n -le 1 ]; then
+  n=100
+fi
+file_path="$2"
+if [ -z "$2" ]; then
+  file_path='/tmp/test.log'
+fi
+
+echo "Emitting $n logs to $file_path"
+
+for i in $(seq 1 $n); do
+  echo "$(date -Is) [foo] This is a log line" >>$file_path
+  echo "$(date -Is) [bar] This is a log line" >>$file_path
+  sleep 1 # sleep so the timestamps for the logs are different
```

Comment:
Just to make sure this is 1ms right? If 1s I feel 100 iteration is a bit overkill.

Reply:
No, `sleep` is 1 second. 🤔  I went back and forth on this. I could include the count in the log line itself, like `This is the $i log line`. The original intention was to make sure that the log lines were a little unique, and I was relying on the timestamp for that. I'm open to ideas.

Reply:
see https://github.com/aws/amazon-cloudwatch-agent/pull/436#discussion_r846491204 I'm just going to redo this in Golang. I originally had this as a shell script just cause it was quick and easy to test out on a fresh host, but writing the log lines natively in Golang is probably the better approach.